client/src/components/Video.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import "./Video.css";
+import { AiOutlineLike, AiOutlineDislike, AiOutlineDelete } from 'react-icons/ai';
+
+const Video = ({ id, title, url, rating, onDelete }) => {
+  
+  const [LiveRating, setLiveRating] = useState(rating); 
+  
+  
+  // const handleLike = () => {
+  //   if (id) {
+  //     setLiveRating(LiveRating + 1);
+  //     updateRating(LiveRating + 1);
+
+  //   }
+  // };
+
+  // const handleDislike = () => {
+  //   if (id && LiveRating > 0) {
+  //     setLiveRating(LiveRating - 1);
+  //     updateRating(LiveRating - 1);
+  //   }
+  // };
```

Comment:
I would deleted old commented code, just causes mess, you can always find it in your git history if you want to find it again.